ruby_programming/object_oriented_programming_basics/project_tic_tac_toe.md

```diff
@@ -9,8 +9,10 @@ Build a tic-tac-toe game on the command line where two human players can play ag
 <div class="lesson-content__panel" markdown="1">
   1. Think about how you would set up the different elements within the game... What should be a class? Instance variable?  Method? A few minutes of thought can save you from wasting an hour of coding.
   2. Build your game, taking care to not share information between classes any more than you have to.
-  3. Post your solution below, then check out the example solution provided.
+  3. Post your solution below, then check out the example solution below from RosettaCode.org.
```

Comment:
Sorry if I wasn't clear

I meant we'd move the link directly into point 3 like...

`3. Post your solution below, then check out the example solution from [RosettaCode.org](http://rosettacode.org/wiki/Tic-tac-toe#Ruby).`

Reply:
Oh, I did try that but the markdown didn't seem to work because it was inside the class. I wasn't sure how to do it without keeping the link outside of the class. It didn't highlight correctly in the MD file, What am I missing?

Reply:
```suggestion
  3. Post your solution below, then check out the example solution from [RosettaCode.org](http://rosettacode.org/wiki/Tic-tac-toe#Ruby).
```

Reply:
If you are ever in doubt, use the [Lesson Markdown Preview Tool!](https://www.theodinproject.com/lessons/preview).

Reply:
Oh I used the VSCode tool. It showed no link. My mistake.